src/cmd/build.js

```diff
@@ -56,8 +100,19 @@ async function defaultPackageCreator(
     filter: (...args) => fileFilter.wantFile(...args),
   });
 
+  let messageFile: string;
+  let extensionName: string = manifestData.name;
+
+  if (manifestData.default_locale) {
+    messageFile = path.join(sourceDir, '_locales',
+                                    manifestData.default_locale,
```

Comment:
nit: this only needs to be indented two spaces unless you want to align it to the opening parenthesis of `path.join(` for readability.